docs/metrics.md

```diff
@@ -124,39 +109,44 @@ Services.telemetry.scalarSet(
 	"category.scalar_name", value
 );
 ```
-e.g. `lockboxV1.datastoreCount` for the scalar name.
+e.g. `lockbox-addon.loginCount` for the scalar name.
 
 We can also use `scalarAdd` to increment a scalar value by some amount.
 
 ## Scalar Metrics
 
-These are the metrics we currently collect regarding the state of user datastores.
+These are the metrics we currently collect regarding the state of the user's login storage.
+
+- `loginCount` (SCALAR_TYPE_COUNT). Current count of the number of items in the user's login storage. If possible, this scalar should be set upon opening either the doorhanger or full extension interface.
+
+- `loginsAccessedAllTime` (SCALAR_TYPE_COUNT). Running total of logins accessed (copied, revealed) through the extension interface or doorhanger. Should never be reset.
+
+- `loginsAccessed` (SCALAR_TYPE_COUNT). Total of logins accessed (copied, revealed) since last ping through the extension interface or doorhanger. TODO: in native code scalars are automatically reset when a new ping is cut. Will that happen for us?
```

Comment:
This should be easy on a per-session basis.

I believe the pings should be reset the same way as native code, since the `browser.telemetry` APIs are just a passthrough to the `Services.telemetry` APIs. @georgf might know for sure.

Reply:
That’s correct, they reset per ping/subsession.

The semantics for the web extension Telemetry API are the same, it’s just a different way to access them.

Reply:
Thanks for the clarification Georg. I'm going to remove the All-time scalar for now, this is relatively straightforward to compute on a per-client basis. It also mitigates some of the anonymity issues discussed with Jared.